waspc/data/Generator/templates/sdk/server/crud/_operationTypes.ts

```diff
@@ -0,0 +1,105 @@
+{{={= =}=}}
```

Comment:
We generate server crud types in the SDK so they can be used for full-stack type safety.